vgc/pipeline.py

```diff
@@ -27,6 +29,13 @@ def start(self):
         self.view_controller.start()
         self.input_adapter.start()
 
+
+    def set_cromping_point(self, phi, theta):
+        """" Sets the point for the filter to crop out """
+        # BUG: THIS NEED TO BE FIXED!  what should zoom be???
+        self.camera_filter.update(jaw_in=theta, pitch_in=phi, zoom_in=2)
```

Comment:
Is it supposed to be "yaw"? I'd rather use my jaw for chewing food then flying a drone :p